kolibri_explore_plugin/assets/src/components/InstallContentModal.vue

```diff
@@ -0,0 +1,230 @@
+    >
+      <template #modal-title>
+        Welcome to Endless Learning
+      </template>
+      <h2 v-if="downloading">
+        Downloading&hellip;
+      </h2>
+      <h2 v-else>
+        Download Endless Learning Collection
+      </h2>
+      <p v-if="downloading">
+        You can close this window during the download and continue to
+        explore the existing content.
+      </p>
+
+      <div v-if="downloading" class="d-block text-center">
+        <b-progress
+          :value="progress"
+          :max="100"
+          showProgress
+          animated
+          class="my-2"
+        />
+        <b-button v-b-toggle.details variant="link">
+          More details
+        </b-button>
+        <b-collapse id="details" class="mt-2">
+          <div v-for="job in filteredJobs" :key="job.id">
+            <b-row>
+              <b-col>{{ job.channel_name }}</b-col>
+              <b-col cols="9">
+                <div v-if="job.status === 'COMPLETED'">
+                  Completed
+                </div>
+                <b-progress
+                  v-else
+                  :value="job.percentage"
+                  :max="1"
+                  variant="success"
+                  showProgress
+                  animated
+                />
+              </b-col>
+            </b-row>
+          </div>
+        </b-collapse>
+      </div>
+      <template #modal-footer>
+        <b-button
+          class="mt-3"
+          block
+          @click="downloadContent"
+        >
+          Download
+        </b-button>
+      </template>
+    </b-modal>
+  </div>
+
+</template>
+
+
+<script>
+
+  import axios from 'axios';
+  import urls from 'kolibri.urls';
+  import { mapState } from 'vuex';
+
+  import DownloadIcon from 'vue-material-design-icons/ProgressDownload.vue';
+  import { ChannelResource } from 'kolibri.resources';
+  import { showChannels } from '../modules/topicsRoot/handlers';
+
+  export default {
+    name: 'InstallContentModal',
+    components: { DownloadIcon },
+    data() {
+      return {
+        pollingId: null,
+        jobs: null,
+        isModalVisible: false,
+        channelsDownloaded: 0,
+        downloading: false,
+      };
+    },
+    computed: {
+      ...mapState(['visibleInstallContentModal']),
+      filteredJobs() {
+        if (!this.jobs) {
+          return [];
+        }
+        return this.jobs.filter(j => j.percentage <= 1);
+      },
+      progress() {
+        const jobs = this.filteredJobs.map(j => j.percentage);
+        const sum = jobs.reduce((a, b) => a + b, 0);
+        return (100 * sum) / this.filteredJobs.length;
+      },
+      buttonStyles() {
+        const stop = this.progress;
+        const color2 = 'rgba(255, 255, 255, 0.2)';
+        const color1 = 'rgba(255, 255, 255, 0)';
+        return {
+          background:
+            'linear-gradient(90deg,' +
+            `${color1} 0%, ${color1} ${stop}%,` +
+            `${color2} ${stop}%, ${color2} 100%)`,
```

Comment:
there are better ways to do this than hardcoding the CSS, but it's totally fine for an MVP